Cython/Compiler/Parsing.py

```diff
@@ -2763,10 +2767,10 @@ def looking_at_expr(s):
 
         dotted_path.reverse()
         for p in dotted_path:
-            s.put_back(u'IDENT', p)
-            s.put_back(u'.', u'.')
+            s.put_back(u'IDENT', *p)
+            s.put_back(u'.', u'.', p[1])  # gets the position slightly wrong
```

Comment:
We could store the dot positions in a separate list and zip both lists together here. Not very costly, I'd say.

Reply:
I'd like to ignore this comment specifically because (a) it's only a small off-by-one error, and (b) I'm pretty sure I can get rid of this whole `looking_at_expr` function as a next step. So it doesn't really some worth fixing here.